src/components/App/index.js

```diff
@@ -163,25 +164,12 @@ export class App extends Component {
 			return language;
 		};
 
-		const browserLanguage = () => (navigator.userLanguage || navigator.language);
-
-		const normalizeLanguageString = (languageString) => {
-			let [languageCode, countryCode] = languageString.split ? languageString.split(/[-_]/) : [];
-			if (!languageCode || languageCode.length !== 2) {
-				return 'en';
-			}
-			languageCode = languageCode.toLowerCase();
-
-			if (!countryCode || countryCode.length !== 2) {
-				countryCode = null;
-			} else {
-				countryCode = countryCode.toUpperCase();
-			}
-
-			return countryCode ? `${ languageCode }_${ countryCode }` : languageCode;
+		const iframeLanguage = () => {
+			const { iframe: { language } = {} } = this.props;
+			return language;
 		};
 
-		I18n.changeLocale(normalizeLanguageString(configLanguage() || browserLanguage()));
+		I18n.changeLocale(normalizeLanguageString(iframeLanguage() ? iframeLanguage() : configLanguage() || browserLanguage()));
```

Comment:
Move this code into `updateWidgetLanguage` method.

Reply:
ok